### ISSUE TITLE
Language code for Ukranian missing

### ISSUE THREAD

**Issue by msche** [DeepLcom/deepl-java#59]
I was creating a mapping for our translation service and noted that the LanguageCode for Ukranian (UK) was not defined in the class. Does this mean the language is not supported?

I'm currently using 1.7.0

**Comment by JanEbbing** [comment]
Good catch! Ukrainian is supported by the API and you should be able to use it anyway with 1.7.0 by calling e.g.

```
String authKey = "f63c02c5-f056-...";  // Replace with your key
        translator = new Translator(authKey);
        TextResult result =
                translator.translateText("Hello, world!", null, "UK");
```

but of course it should be in the `LanguageCodes`. I'll add the missing language(s) and check all other libraries as well, this must've gotten overlooked during the original language release.

**Comment by msche** [comment]
Hi Jan, Is the language code already added?

**Comment by JanEbbing** [comment]
Sorry, I thought I had added this already. I made a merge request to add it, thanks for the reminder :)

**Comment by JanEbbing** [comment]
This is available in 1.10.0 now.
